tests/Inventory.spec.ts

```diff
@@ -0,0 +1,816 @@
+import { expect, test } from "@playwright/test";
+
+// Use the authenticated state
+test.use({ storageState: "tests/.auth/user.json" });
+
+/**
+ * Test data generator for pharmacy-related entities
+ */
+function generatePharmacyTestData() {
+  const timestamp = Date.now();
+  return {
+    productKnowledge: {
+      name: `Test Med ${timestamp}`,
+      slug: `test-med-${timestamp}`,
+      type: "medication",
+      category: "medicine",
+      description: "Test medicine for automated testing",
+      baseUnit: "tablet",
+      dosageForm: "tablet",
+    },
+    chargeItemDefinition: {
+      title: `Medi ci ${timestamp}`,
+      slug: `Medi-ci-${timestamp}`,
+      basePrice: "100.00",
+      mrp: "120.00",
+      purchasePrice: "80.00",
+      description: "Test charge item for automated testing",
+    },
+    requestOrder: {
+      internal: {
+        name: `Internal Order ${timestamp}`,
+        note: "Test internal order for automated testing",
+        priority: "routine",
+        reason: "stock-replenishment",
+      },
+      external: {
+        name: `External Order ${timestamp}`,
+        note: "Test external order for automated testing",
+        priority: "routine",
+        reason: "stock-replenishment",
+        supplier: "Test Supplier",
+      },
+    },
+  };
+}
+
+test.describe("Inventory Management", () => {
+  let facilityId: string;
+  let testData: ReturnType<typeof generatePharmacyTestData>;
+
+  test.beforeEach(async ({ page }) => {
+    testData = generatePharmacyTestData();
+
+    // Navigate to home page (user is already authenticated)
+    await page.goto("/");
+
+    // Navigate to a facility with inventory capabilities
+    await page
+      .getByRole("link", { name: /facility with patients/i })
+      .first()
+      .click();
+
+    // Extract facility ID from URL
+    const url = page.url();
+    const facilityMatch = url.match(/\/facility\/([^/]+)/);
+    facilityId = facilityMatch ? facilityMatch[1] : "";
+    expect(facilityId).toBeTruthy();
+  });
+
+  test.describe("Product Knowledge Management", () => {
+    test("should create a new product knowledge entry", async ({ page }) => {
+      // Step to manage category - check if "Test Category" exists
+      await test.step("Search for Test Category", async () => {
+        await page.getByRole("button", { name: "Toggle Sidebar" }).click();
+        await page
+          .getByRole("button", { name: "Settings", exact: true })
+          .click();
+        await page.getByRole("link", { name: /product knowledge/i }).click();
```

Comment:
Oh no, we are mixing concerns here. This should be a completely different test file altogether. 

`/facility/settings/productKnowledge.spec.ts`